gateway/src/apicast/cli/command/start.lua

```diff
@@ -44,10 +44,13 @@ local function update_env(env)
     end
 end
 
-local function nginx_config(context, dir, path, env)
-    update_env(env)
 
-    local template = Template:new(context, dir, true)
+local function apicast_root()
+    return resty_env.get('APICAST_DIR') or pl.path.abspath('.')
```

Comment:
Should we call `.value` instead?

Reply:
Yes. Good point!

Reply:
Did mass change in https://github.com/3scale/apicast/pull/496/commits/569c52061498256d549b2ad9f9d47673c8cabfc5 and will squash it.